can/notifier.py

```diff
@@ -13,57 +13,62 @@
 
 class Notifier(object):
```

Comment:
We should probably add a test for this class that includes multiple buses & listeners.